src/org/parosproxy/paros/core/scanner/NameValuePair.java

```diff
@@ -32,6 +32,10 @@
     public static final int TYPE_COOKIE = ScannerParam.TARGET_COOKIE;
     public static final int TYPE_HEADER = ScannerParam.TARGET_HTTPHEADERS;
     public static final int TYPE_POST_DATA = ScannerParam.TARGET_POSTDATA;
+    public static final int TYPE_MULTIPART_DATA_PARAM = 33;

```

Comment:
Worth adding a since JavaDoc tag, also, mention to which part of the content is the type referring to? (Same comment for the other types.)

Reply:
Will do